crescent/bot.py

```diff
@@ -150,7 +150,12 @@ def include(self, command: META_STRUCT | None = None):
         return command
 
     @staticmethod
-    def print_banner(banner: Optional[str], allow_color: bool, force_color: bool) -> None:
+    def print_banner(
+        banner: Optional[str],
+        allow_color: bool,
+        force_color: bool,
+        extra_args: Optional[Dict[str, str]] = None,
+    ) -> None:
         print_banner(banner, allow_color, force_color)
```

Comment:
Missing extra_args?

Reply:
I ignored it because extra_args is set in the `_ux.py` file. I did it like this to reduce clutter in `bot.py`

Reply:
i changed the code so it might be simpler